telemetry.go

```diff
@@ -25,6 +25,10 @@ var (
 		},
 		[]string{"type"},
 	)
+	eventsMissCount = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "statsd_exporter_events_misses_total",
```

Comment:
I'd go for `statsd_exporter_unmapped_events_total`, easier to understand what it's about.